cpp/src/strings/regex/regex.inl

```diff
@@ -127,6 +127,16 @@ __device__ __forceinline__ void reprog_device::reljunk::swaplist()
   list2    = tmp;
 }
 
+/**
+ * @brief Check for supported new-line characters
+ *
+ * '\n, \r, \u0085, \u2028, or \u2029'
+ */
+__device__ __forceinline__ bool is_newline(char32_t const ch)
```

Comment:
Not sure if this should be a permanent change. For java this is very useful, but Python compatibility would probably dictate this be configurable behind a flag.

Reply:
Thanks. I just need you tell me if it works for Spark and does it improve performance.